_posts/blog/2018-19-10-Reseña-Digital.md

```diff
@@ -0,0 +1,31 @@
+Reseña del archivo digital "Archivo Mapuche Warria Mew. Waria ñi tukulpan dungu/ Ampliando los horizontes de la memoria."
```

Comment:
Falta el YAML Header. Instrucciones: http://www.halperta.com/criticalarchives/tutorial/writing-in-markdown/